sklearn/svm/tests/test_bounds.py

```diff
@@ -74,3 +76,44 @@ def test_ill_posed_min_c():
 def test_unsupported_loss():
     with pytest.raises(ValueError):
         l1_min_c(dense_X, Y1, loss='l1')
+
+
+# TODO test deterministic results on differing systems
+# def test_set_seed():
+#     def _test(seed, val):
+#         # TODO currently causes 99% coverage in test_bounds.py
+#         if seed is not None:
+#             set_seed_wrap(seed)
+#         x = bounded_rand_int_wrap(100)
+#         assert(x == val), 'Expected {} but got {} instead'.format(val, x)
+
+#     # TODO should be default seed for std::mt19937,
+#     # but different results on jupyter lab
+#     # _test(None, 81)
+#     _test(5489, 81)  # default seed for std::mt19937
+#     _test(0, 54)
+#     _test(4294967295, 9)  # max unsigned int size
+
+
+_MAX_INT = 2147483647
+
+
+@pytest.mark.parametrize('orig_range, n_pts',
+                         [(_MAX_INT, 10000), (100, 10)])
+def test_bounded_rand_int(orig_range, n_pts):
+    n_iter = 100
+    ks_pvals = []
+    uniform_dist = stats.uniform(loc=0, scale=orig_range)
+    # perform multiple samplings to make chance of outlier sampling negligible
+    for _ in range(n_iter):
+        # Deterministic random sampling
+        sample = [bounded_rand_int_wrap(orig_range) for _ in range(n_pts)]
+        res = stats.kstest(sample, uniform_dist.cdf)
+        ks_pvals.append(res.pvalue)
+    min_10pct_pval = np.percentile(ks_pvals, q=10)
+    # lower 10th quantile pvalue <= 0.05 means that the test rejects the
+    # null hypothesis that the sample came from the uniform distribution
+    assert(min_10pct_pval > 0.05),\
+        'lower 10th quantile p-value of {} not > 0.05, '\
+        'therefore distribution isn\'t sufficiently uniform'\
+        .format(min_10pct_pval)
```

Comment:
From my previous comment in the review - I added a p-vals uniformity test and a comment

```suggestion
    # Under the null hypothesis, p-values should be uniformly distributed.
    # (this may seem counter-intuitive but is backed by multiple refs)
    # So we can do two checks: 
    
    # (1) check uniformity of p-values 
    uniform_p_vals_dist = stats.uniform(loc=0, scale=1)
    res_pvals = stats.kstest(ks_pvals, uniform_p_vals_dist.cdf)
    assert res_pvals.pvalue > 0.05
    
    # (2) (safety belt) check that most p-values are above 0.05
    min_10pct_pval = np.percentile(ks_pvals, q=10)
    # lower 10th quantile pvalue <= 0.05 means that the test rejects the
    # null hypothesis that the sample came from the uniform distribution
    assert(min_10pct_pval > 0.05),\
        'lower 10th quantile p-value of {} not > 0.05, '\
        'therefore distribution isn\'t sufficiently uniform'\
        .format(min_10pct_pval)
```